TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/DemoTeleOp.kt

```diff
@@ -0,0 +1,35 @@
+package org.firstinspires.ftc.teamcode.opmode.teleop
+
+import com.qualcomm.robotcore.eventloop.opmode.Disabled
+import com.qualcomm.robotcore.eventloop.opmode.OpMode
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp
+import org.firstinspires.ftc.teamcode.api.DemoAPI
+import org.firstinspires.ftc.teamcode.api.linear.DemoLinearAPI
+
+@TeleOp(name = "Demo TeleOp")
+@Disabled
+class DemoTeleOp : OpMode() {
+    // Run once, when "Init" is pressed on driver hub.
+    override fun init() {
+        // Initializes an API, shared code used by components.
+        DemoAPI.init(this)
+
+        // Initialize the April Vision API
+        DemoLinearAPI.init(this)
```

Comment:
This line will throw an exception because linear APIs cannot be initialized with `OpMode`s, only `LinearOpMode`s.

Could you either remove it or comment this line out, explaining why it won't work?